### PR TITLE
Added support for Niceboy sensors

### PR DESCRIPTION
**Pull request by vsigler**
Added support for a few Niceboy sensors:
https://niceboy.eu/en/product/orbis-motion-sensor
https://niceboy.eu/en/product/orbis-vibration-sensor
https://niceboy.eu/en/product/orbis-water-sensor
https://niceboy.eu/en/product/orbis-wd-sensor

Images added in PR here: https://github.com/Koenkk/zigbee2mqtt.io/pull/2147

All of those are TuYa-based, just needed fingerprints and whitelabel markers added.

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -540,6 +540,7 @@ const definitions: Definition[] = [
             {vendor: 'TuYa', model: 'iH-F001'},
             {vendor: 'Tesla Smart', model: 'TSL-SEN-DOOR'},
             {vendor: 'Cleverio', model: 'SS100'},
+            tuya.whitelabel('Niceboy', 'ORBIS Windows & Door Sensor', 'Door sensor', ['_TZ3000_qrldbmfn']),
```
Comment (Koenkk) [inline src/devices/tuya.ts:543]: Instead of `ORBIS Windows & Door Senso` use the model number which is written on the device.

**Comment by Koenkk** [comment]
Note to self; after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/2147/files

File: src/devices/tuya.ts (hunk 1 above)
Comment (vsigler) [reply src/devices/tuya.ts:543]: It is actually what is listed in both the manual and on the device itself as "model". I did not find any other number that I could use. It is the same for all of the sensors.

**Comment by Koenkk** [comment]
I see, thanks!

**Pull request merged**